test/docker/compose.go

```diff
@@ -571,8 +571,8 @@ func (d *Compose) Start(ctx context.Context) (*DockerCompose, error) {
 		}
 		containers = append(containers, container)
 	}
-	if d.withPaLMApiKey != "" {
-		envSettings["PALM_APIKEY"] = d.withPaLMApiKey
```

Comment:
just to confirm that when the environment variable with the legacy name is set shouldn't we set `envSettings["GOOGLE_APIKEY"] = d.withPaLMApiKey`

Reply:
I think that we do it rn, check the PR. I have merged a PR that renames the text2vec-palm module and that change was there